src/demo-app/demo-app/demo-app.scss

```diff
@@ -1,3 +1,19 @@
+// The default indigo-pink theme will load through the index file. This makes the SCSS build faster
+// since Sass doesn't need to build a "prebuilt" theme again.
+@import '../../lib/core/theming/all-theme';
```

Comment:
We should try to keep our demo app as close as possible to how user apps would be configured. So I agree that we should avoid drilling down into the lib folder.  We should be able to use the top level `@angular/material` import if we tweak the `includePaths` option in our sass build config to include the dist folder.